documentation/Installing-Clear-Containers-on-Ubuntu.md

```diff
@@ -11,14 +11,19 @@ used but some issues may exist.
 This step is optional and required only in the case where docker is not installed
 on the system, or the explicitly supported version of docker is required.
 
+Note that `xenial` (Ubuntu 16.04) is specified below. This is required
+for both Ubuntu 16.04 *and* Ubuntu 16.10 (`yakkety`) since the Clear
+Containers supported version of Docker is not available as an Ubuntu
+16.10 package.
+
 ```
 sudo groupadd docker
 sudo -E gpasswd -a $USER docker
 sudo apt-get install -y apt-transport-https ca-certificates
 curl -fsSL https://yum.dockerproject.org/gpg | sudo apt-key add -
-sudo add-apt-repository "deb https://apt.dockerproject.org/repo/ ubuntu-$(lsb_release -cs) main"
+sudo add-apt-repository "deb https://apt.dockerproject.org/repo/ ubuntu-xenial main"
```

Comment:
not sure, but probably you should replace this line too
https://github.com/jodh-intel/cc-oci-runtime/commit/c228baae406bb77b79fd25312490d2bd579814ea#diff-4a4bb8ef2d79f51c239fbf74acda82b5R28